api/resources/translations/messages-es.properties

```diff
@@ -372,180 +372,189 @@ android_app.data_usage.rx = Recibido
 android_app.data_usage.system.title = En todo el sistema
 android_app.data_usage.title = Uso de datos de la aplicación
 android_app.data_usage.tx = Transmitido
-android_app.version.title = Versión de Android app
-app.name =
-app.version.unknown = Desconocido - conexión a internet requerida.
+android_app.os_android_version.title = Android OS
+android_app.package_name.title = nombre del paquete de la aplicación de Android
+android_app.version.title = versión de la aplicación de Android
+android_app.version_code.title = código de versión de la aplicación de Android
+app.name = Aplicación
+app.version.unknown = Desconocido - se requiere conexión a Internet.
 associated.contact = Contacto asociado
-associated.contact.help = Cuando este usuario crea informes, será asignado a este contacto
-autoreply = autorespuesta
-birth_date = Fecha del parto
-branding =
-branding.favicon.field = 
-branding.logo.field =
-branding.title.field = 
-branding.title.field.help =
+associated.contact.help = Cuando este usuario crea informes, serán asignados a este contacto.
+autoreply = respuesta automática
+birth_date = fecha de nacimiento
+branding = marca
+branding.favicon.field = Icono pequeño
+branding.favicon.field.help = Este icono se muestra en la pestaña del navegador. Debe ser un archivo .ico de 32x32 píxeles.
+branding.logo.field = Logo
+branding.logo.field.help = Se muestra en la página de inicio de sesión y en dispositivos con pantalla grande.
+branding.title.field = Título
+branding.title.field.help = Este texto se mostrará en la pestaña del navegador y como título del PWA.
+branding.icon.field = Icono grande
+branding.icon.field.help = Se mostrará para instalaciones de PWA. Debe ser al menos 144x144 píxeles.
 browser.compatibility.title = Contacte a su supervisor
 browser.compatibility.description = Por favor contacte a su supervisor para actualizar su navegador.
 browser.compatibility.confirm = Ok
 bulkdelete.confirm.action = Eliminar
 bulkdelete.confirm.title = ¿Eliminar el registro?
 bulkdelete.confirm.title.plural = ¿Eliminar registros seleccionados?
 call = Llamar
-case_id =
-child_birth_date = Fecha nacimiento niño
-child_birth_outcome = Resultado del parto
+case_id = Identificación del caso
+child_birth_date = Fecha de nacimiento del niño
+child_birth_outcome = Resultado del nacimiento del niño
 child_birth_weight = Peso del niño al nacer
-clear = borrar
-cleared = Marca de Tiempo Borrada
-clientDdoc.version = Versión de Cliente DDoc
+clear = Borrar
+cleared = Fecha y hora del borrado
+clientDdoc.version = Revisión del webapp en el cliente
```

Comment:
Why `Revisión` and not `Versión`?

Reply:
The English file says: `Client webapp revision`
I kept it similar since I think it's a different form version. I think this is the replication sequence that tell us if the user device is up-to-date with the server.

Reply:
I believe it's best to keep the original word `revision` to match the English file.